lib/events/dynamoevents/dynamoevents.go

```diff
@@ -287,13 +287,11 @@ func New(ctx context.Context, cfg Config) (*Log, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	// create DynamoDB service:
+	// Create DynamoDB service.
 	svc, err := dynamometrics.NewAPIMetrics(dynamometrics.Events, dynamodb.New(b.session, &aws.Config{
 		// Setting this on the individual service instead of the session, as DynamoDB Streams
 		// and Application Auto Scaling do not yet have FIPS endpoints in non-GovCloud.
 		// See also: https://aws.amazon.com/compliance/fips/#FIPS_Endpoints_by_Service
-		// TODO(reed): This can be simplified once https://github.com/aws/aws-sdk-go/pull/5078
-		// is available (or whenever AWS adds the missing FIPS endpoints).
```

Comment:
Yeah, I purposely left this one alone, as otherwise, we have to explicitly disable use of FIPS endpoints instead of just making it neutral, which I felt would actually be a regression.

I'm not tied to this other format, so I can just remove the TODOs entirely.

Reply:
TODOs removed.